go-controller/pkg/types/const.go

```diff
@@ -100,6 +100,9 @@ const (
 	GatewayLBUDP          = "UDP_lb_gateway_router"
 	GatewayLBSCTP         = "SCTP_lb_gateway_router"
 
+	// Loadbalancer options
+	OptionsHairpin = "options:hairpin_snat_ip=" + V4LBHairpinMasquradeIP + " " + V6LBHairpinMasquradeIP
```

Comment:
maybe change the variable name to something more readable, like LBHairpinOptions

Reply:
Yep sounds good